tower/src/balance.rs

```diff
@@ -0,0 +1 @@
+pub use tower_balance::*;
```

Comment:
Is it preferable to have module with glob re-exports, versus `pub extern crate tower_balance as balance`?

Reply:
I don't care too much, just seemed like a re-export of the whole crate was easier. :shrug: